src/components/dialogs/parameters/security-analysis/security-analysis-parameters.tsx

```diff
@@ -123,12 +164,19 @@ export const SecurityAnalysisParameters: FunctionComponent<{
         [params]
     );
 
-    const { handleSubmit, formState } = formMethods;
+    const { handleSubmit, formState, reset } = formMethods;
 
-    const updateLimitReductions = useCallback(
+    const updateSAParameters = useCallback(
         (formLimits: Record<string, any>) => {
```

Comment:
TODO maybe later : type this formLimits object since we now know what to expect according to yup schema